gruntfile.js

```diff
@@ -1,4 +1,39 @@
 module.exports = function (grunt) {
+
+    const aiCoreDefaultNameReplacements = [
+       {
+           src: "./src/JavaScriptSDK.Enums/SdkCoreNames.ts",
+           import: "./src/JavaScriptSDK.Enums/SdkCoreNames"
+       },
+       {
+           src: "./src/JavaScriptSDK.Enums/ConfigEnums.ts",
+           import: "./src/JavaScriptSDK.Enums/ConfigEnums"
+       }
+   ];
+
+   const aiDefaultNameReplacements = [
+        {
+            src: "../shared/AppInsightsCore/src/JavaScriptSDK.Enums/SdkCoreNames.ts",
+            import: "@microsoft/applicationinsights-core-js"
+        },
+        {
+            src: "../shared/AppInsightsCore/src/JavaScriptSDK.Enums/ConfigEnums.ts",
+            import: "@microsoft/applicationinsights-core-js"
+        },
+        {
+            src: "../../shared/AppInsightsCore/src/JavaScriptSDK.Enums/SdkCoreNames.ts",
+            import: "@microsoft/applicationinsights-core-js"
+        },
+        {
+            src: "../../shared/AppInsightsCore/src/JavaScriptSDK.Enums/ConfigEnums.ts",
+            import: "@microsoft/applicationinsights-core-js"
+        }
+    ];
+
+    const aiInternalConstants = [
+        "./src/InternalConstants.ts"
```

Comment:
The location within each project (module) where to find the file(s) that identify constant strings to be dynamically read.